docs/man/containerd-config1.x.md

```diff
@@ -0,0 +1,368 @@
+## Default Configuration: Containerd 1.x:
```

Comment:
these files are specifically CRI plugin config details.. pls name the file accordingly adding -cri- same for the title